verifier/src/main/res/drawable/ic_settings.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Don't we have this already in the common module?

Reply:
same same but different